content/about/faq.md

```diff
@@ -1,23 +1,54 @@
+
+### What is Vite
+
+[Vite (Vite.js)](https://vitejs.dev/) is a lightning fast tool for frontend development. It is replacing [Webpack](https://webpack.js.org/) and will be used to compile styles and scripts as well as bundle files.  
+
+### What is our timeline for transitioning to 11ty?
+
+We expect the bulk of development work to be done by July 2022 with Quire's official open-source launch scheduled for the Fall. We look to the Quire community for feedback and to help us spot bugs that the transition may have generated. The move will be a gradual work in progress, and we appreciate our users' patience.
+
+### How does this affect older or in process Quire projects?
+
+Quire projects built with the Hugo version of Quire, can be manually migrated to the 11ty version as needed by making a handful of global changes to Markdown files and copying over various configuration values. Documentation will be provided and our team is available for support. You can also install an earlier version of Quire with `npm install @thegetty/quire-cli@0.20.2` to run older Hugo-based projects, but that version will only be supported for a limited time.
+
+Let us know if you will be starting any new Quire projects as early as June 2022. It will be advantageous to adopt the 11ty version of Quire early, even though our implementation will still be underway. We are happy to work closely with you to troubleshoot your use of Quire during this transition period. Reach out to us at [quire@getty.edu](mailto:quire@getty.edu).
+
+### Will there be new documentation?  
+
+We will be writing new documentation for the 11ty version of Quire. The older Quire documentation will be gradually phased out over time. For the most part, changes to the docs will be fairly minor, especially on the content creator/editor side of things.
+
+We are looking for contributions to help support the creation of developer-centric documentation. If you are interested in contributing please see our [Contributor Guidelines](https://github.com/thegetty/quire-docs/blob/main/CONTRIBUTING.md).
+
+### Can I explore 11ty for myself?
+
+We will have an 11ty branch of Quire available for people to play around with and offer feedback in the coming weeks.
+
+### What are some other Quire developments underway?
+
+In addition to our work on 11ty, we are also working on switching our image viewing from Leaflet to [CanvasPanel](https://canvas-panel.digirati.com/#/about). We are also greatly improving our IIIF capabilities, including adding the ability to annotate images and layer images. We are also working to replace Prince XML with the open-source PDF generator, [Paged.js](https://pagedjs.org/). All this development work is still underway. We thank you for your patience.  
```

Comment:
"including adding the ability to annotate images and layer images" isn't quite right as I think anotations would have to be made outside Quire. Maybe just leave as "including adding the ability to layer images" for now